services/121-service/src/registration/services/registrations-pagination.service.ts

```diff
@@ -133,8 +133,9 @@ export class RegistrationsPaginationService {
     }
 
     if (hasPersonalReadPermission) {
-      paginateConfigCopy.relations = ['data'];
-      paginateConfigCopy.searchableColumns = ['data.(value)'];
+      paginateConfigCopy.relations = ['data', 'dataSearchBy'];
```

Comment:
Can these magic strings also be used from enum-values throughout the back-end?
To keep track of all use-cases automatically? And to prevent typo's, etc.